### DISCUSSION TITLE
Form Filter is needed.

### DISCUSSION THREAD

**Discussion by Tomycl** [icflorescu/mantine-datatable#678] [Q&A]
The current filtering function is on the header, but these take effect in real time. Is there a possibility to make a filter form interface to pass filter items set in batches? Currently, we need to use the parent component to coordinate the filtering form and table components, which is a bit troublesome. Can we have a built-in filter form like pagination?

**Comment by icflorescu** [comment]
I'm sorry, but this is way out of the scope of this library. There are infinite ways to implement this and there's no "one solution to fit all". Even in my own projects I use the parent component for that.
